pom.xml

```diff
@@ -20,6 +20,14 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 	</properties>
 
+    <repositories>
```

Comment:
Why did we decide to switch to this approach?

Reply:
Thanks for taking a look. It is no longer needed. I removed it.